### ISSUE TITLE
Graphic Views are displaying None Level Events on data points

### ISSUE THREAD

**Issue by Joelhaggar** [infiniteautomation/ma-core-public#587]
Previously None level events would not be displayed on the data points for a graphic view.

**Comment by Joelhaggar** [comment]
I think it might display them even after they are acknowledged.

**Comment by terrypacker** [comment]
Interesting piece of information here, the NONE level events are acknowledged when they are saved to the database.  They have the acknowledge timestamp set to the active timestamp.  I found a bug where the UserEventsCache was allowing these to be inserted and they would never be removed because they are never acknowledgeable.

**Comment by Puckfist** [comment]
Hmm. Something is amiss now....

1) None level events have seemingly stopped showing up entirely. I cannot get them to show up on my events pages querying for None level events. I did verify from the SQL console I had raised immense numbers of None events through a 1ms poll.
2) EventManager:147-148 seems weird from how you describe the acknowledging none (which I found in the EventDao, 80-81). Are None and Do Not Log events which could normally be returned to normal bloating this list? My little bit of investigating certainly suggested there were some Do Not Logs in there of very little value, but it seems like it's doing what it should. Just seemed odd.
